packages/connect-core/src/protocol-connect/trailer-mux.ts

```diff
@@ -26,10 +26,24 @@ export function trailerDemux(
     t = new Headers();
   header.forEach((value, key) => {
     if (key.toLowerCase().startsWith("trailer-")) {
-      t.set(key.substring(8), value);
+      t.set(key.replace(/^[Tt]railer-/, ""), value);
```

Comment:
Can't we simply replace the entire if-else block with this line?

Reply:
Unless I'm missing something, the if-else block splits these between two `Headers` objects depending on whether they start with `trailer-`.  If we replaced the if-else block with just this line, then all the keys would be put into the trailer object, right?

Reply:
Silly me, you're right. There's little point in using the regex over the substring call then, I guess.